contentcuration/contentcuration/frontend/shared/languageSwitcher/LanguageSwitcherModal.vue

```diff
@@ -15,15 +15,17 @@
         :layout8="{ span: 4 }"
         :layout12="{ span: 6 }"
       >
-        <KRadioButton
-          v-for="language in languageCol"
-          :key="language.id"
-          v-model="selectedLanguage"
-          :buttonValue="language.id"
-          :label="language.lang_name"
-          :title="language.english_name"
-          class="language-name"
-        />
+        <KRadioButtonGroup>
```

Comment:
In general here I think we are managing these columns a little bit strange :sweat_smile: because on top of this we are using a grid item to render the number of columns, and for each colum we render the languages radio buttons, that means that if we add the KRadioButtonGroup inside the grid item the up and down arrows will just work for one column and not the other.

https://github.com/user-attachments/assets/4165bc10-2fc3-4302-9280-4415b17dfcec

Since KRadioButton should be a direct child of KRadioButtonGroup, I think we should refactor how we manage the column rendering to avoid having two different divs for different columns. I think grid or flex layouts would help here :)

Reply:
Got it. Thanks @AlexVelezLl!